libs/pika/functional/include/pika/functional/detail/function_registration.hpp

```diff
@@ -26,14 +26,7 @@ namespace pika { namespace util { namespace detail {
     template <typename VTable, typename F>
     struct get_function_name_impl
     {
-        static char const* call()
-#ifdef PIKA_HAVE_AUTOMATIC_SERIALIZATION_REGISTRATION
-        {
-            return debug::type_id<F>::typeid_.type_id();
-        }
-#else
-            = delete;
-#endif
+        static char const* call() = delete;
```

Comment:
This whole file could possibly be deleted. Have you checked where it's still used?

Reply:
Indeed it was used in unique_function and function in preprocessor macros that are in fact unused, removing..